### PR TITLE
OP-1167 Add error for unexpected condition on lot management

### PR DESCRIPTION
**Pull request by mwithi**
See OP-1167.

### CODE REVIEW

File: src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java [hunk 1]
```diff
@@ -100,39 +104,41 @@ public class MovStockMultipleDischarging extends JDialog {
...
+	private boolean[] columnEditable = { false, false, false, false, true, false, false, false };
+	private int[] columnWidth = { 50, 100, 70, 50, 70, 50, 100, 80 };
+	private boolean[] columnResizable = { false, true, false, false, false, false, false, false };
 	private boolean[] columnVisible = { true, true, true, true, true, true, !GeneralData.AUTOMATICLOT_OUT, !GeneralData.AUTOMATICLOT_OUT };
- 	private int[] columnAlignment = { SwingConstants.LEFT, SwingConstants.LEFT, SwingConstants.CENTER, SwingConstants.CENTER, SwingConstants.CENTER, SwingConstants.CENTER, SwingConstants.CENTER,
-			SwingConstants.CENTER};
-	private boolean[] columnBold = { false, false, false, false, false, true, false, false};
+	private int[] columnAlignment = { SwingConstants.LEFT, SwingConstants.LEFT, SwingConstants.CENTER, SwingConstants.CENTER, SwingConstants.CENTER,
+			SwingConstants.CENTER, SwingConstants.CENTER,
+			SwingConstants.CENTER };
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java:123]: shouldn't lines 122 and 123 be combined into a single line?

File: src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java [hunk 2]
```diff
@@ -286,6 +292,13 @@ private JTextField getJTextFieldSearch() {
 					} catch (OHServiceException e1) {
 						lots = null;
 						OHServiceExceptionUtil.showMessages(e1);
+						return;
+					}
+					if (lots.isEmpty()) {
+						LOGGER.error("No lots found for medical {} and quantity {}", med.getDescription(), qty);
+						JOptionPane.showMessageDialog(MovStockMultipleDischarging.this,
+										MessageBundle.getMessage("angal.sql.anunexpectederroroccurredpleasecheckthelogs.msg"));
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java:300]: Why isn't this just `MessageDialog.error(this, "angal.sql.anunexpectederroroccurredpleasecheckthelogs.msg")`?

**Review by dbmalkovsky — COMMENTED** [review]
Just some minor comments; otherwise looks OK.

File: src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java (hunk 1 above)
Comment (mwithi) [reply src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java:123]: yes, thanks

File: src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java (hunk 2 above)
Comment (mwithi) [reply src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java:300]: Thanks!

**Pull request merged**